cycode/cli/helpers/tf_content_generator.py

```diff
@@ -0,0 +1,29 @@
+import json
+from typing import List
+
+from cycode.cli.models import ChangeResource
+
+
+def generate_tf_content_from_tfplan(tfplan: str) -> str:
+    planned_resources = _extract_resources(tfplan)
+    return _generate_tf_content(planned_resources)
+
+
+def _extract_resources(tfplan: str) -> List[ChangeResource]:
+    tfplan_json = json.loads(tfplan)
+    resources: List[ChangeResource] = []
+    for change in tfplan_json.get('resource_changes', []):
+        resources.append(
+            ChangeResource(resource_type=change['type'], name=change['name'], values=change['change']['after'])
```

Comment:
is there `after` in case of resource deletion?

Reply:
please change all to safe get

Reply:
We want to stop parsing if the keys are missing, and errors are caught if this is the case